libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
```

Comment:
`bind` does not start the fbo, `begin` does =).

Keeping bind and begin separate might help us in the future, if we need to bind the fbo without setting up matrices, like you would in some more advanced cases.